Unable to build with github.com/shirou/gopsutil/v3 v3.22.10

I am working on packaging this library for use in Debian, and have encountered the following build error when using github.com/shirou/gopsutil/v3 v3.22.10 (the current version of that library and recently updated in Debian):

```
# github.com/minio/madmin-go
src/github.com/minio/madmin-go/health.go:328:36: undefined: host.Warnings
```

It appears that commit https://github.com/shirou/gopsutil/commit/dbc0f20fe38841a0c1509de9dba5084b7c2c09d9 moved `host/types.go` -> `internal/common/warnings.go`.

--- a/health.go
+++ b/health.go
@@ -323,12 +323,7 @@ func GetOSInfo(ctx context.Context, addr string) OSInfo {
 	}
 	osInfo.Info.KernelVersion = kr
 
-	osInfo.Sensors, err = host.SensorsTemperaturesWithContext(ctx)
-	if err != nil {
-		if _, isWarningErr := err.(*host.Warnings); !isWarningErr {
-			osInfo.Error = err.Error()
-		}
-	}
+	osInfo.Sensors, _ = host.SensorsTemperaturesWithContext(ctx)
 
 	return osInfo
 }

